pilot/pkg/model/config.go

```diff
@@ -996,6 +1022,82 @@ func (store *istioConfigStore) AuthenticationPolicyByDestination(hostname string
 	return &out
 }
 
+// Set jwks_uri through openID discovery if it's not set in auth policy.
+func (store *istioConfigStore) SetAuthenticationPolicyJwksURIs(policy *authn.Policy) {
```

Comment:
again, all these logics doesn't need to reference to store. so it should be moved into the authCache struct.

Reply:
done.